libcanard/canard.h

```diff
@@ -285,6 +285,10 @@ struct CanardTxQueueItem
     /// Frames whose transmission deadline is in the past shall be dropped.
     CanardMicrosecond tx_deadline_usec;
 
+    /// Amount of memory allocated for the whole this item, including the payload frame.
+    /// In use to deallocate the item by passing this value to the memory manager (`memFree`).
```

Comment:
```suggestion
    /// The amount of memory allocated for this item, including the frame payload.
    /// This value is needed for memory deallocation because the deallocation function takes the fragment size.
```